src/component/detailed_timer/mod.rs

```diff
@@ -354,46 +354,64 @@ impl Component {
     /// component and their current values.
     pub fn settings_description(&self) -> SettingsDescription {
         SettingsDescription::with_fields(vec![
-            Field::new("Background".into(), self.settings.background.into()),
+            Field::new(
+                "Background".into(),
+                "The background shown behind the component.".into(),
+                self.settings.background.into(),
+            ),
             Field::new(
                 "Timing Method".into(),
+                "Specifies the timing method to use. If not specified, the current timing method of the timer is used for showing the time.".into(),
                 self.settings.timer.timing_method.into(),
             ),
             Field::new(
                 "Comparison 1".into(),
+                "The first comparison to show the segment time of. If it's not specified, the current comparison is used.".into(),
                 self.settings.comparison1.clone().into(),
             ),
             Field::new(
                 "Comparison 2".into(),
+                "The second comparison to show the segment time of. If it's not specified, the current comparison is used, unless the first comparison is also None. This is not shown if the second comparison is hidden.".into(),
```

Comment:
> This is not shown if the second comparison is hidden.

I still see this setting in LiveSplit One even when I enable "Hide Second Comparison". Is that a bug? (Seems similar to https://github.com/LiveSplit/livesplit-core/issues/802)